server/app/controllers/admin/AdminProgramController.java

```diff
@@ -97,7 +101,9 @@ public Result create(Request request) {
             program.getLocalizedDisplayName(),
             program.getLocalizedDisplayDescription(),
             program.getExternalLink(),
-            program.getDisplayMode());
+            program.getDisplayMode(),
+            program.getIsCommonIntakeForm(),
```

Comment:
Instead of passing the feature flag down into the program logic, what if we just override the value of `getIsCommonIntakeForm()` to `false` here and below? Maybe pulling the override out into a shared method.

Reply:
I would be down to do that; my main hesitation is that there's a note in the docs about best practices being to always check feature flags before writing to the DB even if the corresponding UI option is already hidden by a flag and the ProgramService is the closest point to the DB where that makes sense to do. That said, I agree this is a little ugly.

Reply:
Ok I see, cause you need to check the feature flag in ProgramServiceImpl.createProgramDefinition() before you write to the DB.

My preference would've been to @Inject the FeatureFlags into `ProgramServiceImpl`, instead of passing the value down, but it looks like other services also have the value passed down from the controller, which might be to make testing simpler:
https://github.com/civiform/civiform/blob/1b537b040f39d4e9089fdc5f87a8d0a24d4e9b02/server/app/services/applicant/ApplicantService.java#L301